dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemHelpdeskStrategy.java

```diff
@@ -45,9 +51,9 @@ public RequestItemHelpdeskStrategy() {
     @NonNull
     public List<RequestItemAuthor> getRequestItemAuthor(Context context, Item item)
             throws SQLException {
-        boolean helpdeskOverridesSubmitter = configuration
+        boolean helpdeskOverridesSubmitter = configurationService
             .getBooleanProperty("request.item.helpdesk.override", false);
-        String helpDeskEmail = configuration.getProperty("mail.helpdesk");
+        String helpDeskEmail = configurationService.getProperty("mail.helpdesk");
```

Comment:
Small thing, but you created `P_` variables above and then didn't use them in this class.  I see you use them in the tests, but maybe we should use them on lines 55 & 56 as well?

Reply:
This code was removed in [f6ab8ba](https://github.com/DSpace/DSpace/pull/8319/commits/f6ab8ba1fbd42e0c1e77e1c2779c684eed0fc69a).